runtime/syntax/mom.vim

```diff
@@ -0,0 +1,16 @@
+" VIM syntax file
+" Language:	nroff
+" Maintainer:	John Marshall <jmarshall@hey.com>
+" Previous Maintainer:	Pedro Alejandro López-Valencia <palopezv@gmail.com>
+" Previous Maintainer:	Jérôme Plût <Jerome.Plut@ens.fr>
+" Last Change:	2021 Mar 28
```

Comment:
Last change header for a new filetype looks strange

Reply:
Removed